plan.go

```diff
@@ -247,40 +252,28 @@ func extractSelectionSet(ctx *PlanningContext, insertionPoint []string, parentTy
 					continue
 				}
 				implementationType := ctx.Schema.Types[implementationName]
-				possibleId := &ast.InlineFragment{
-					TypeCondition: implementationName,
-					SelectionSet: []ast.Selection{
-						&ast.Field{
-							Alias:      "_id",
-							Name:       "id",
-							Definition: implementationType.Fields.ForName("id"),
-						},
-					},
-					ObjectDefinition: implementationType,
+
```

Comment:
This adds the best-practice from [below](https://github.com/movio/bramble/pull/94/files#r744360598) where the type is checked for an ID definition – I suppose the wild cards here are namespaces, which are just merged scopes without ids?